### PR TITLE
Admin UI: Show Messages of Identity in the Identity details page

### PR DESCRIPTION
**Pull request by JLSRKonk**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [x] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

# Screenshots

ReceivedMessages
SentMessages1
SentMessages2
SentMessagesDialog

### CODE REVIEW

File: AdminUi/src/AdminUi/ClientApp/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -10,13 +10,21 @@ import { IdentityDetailsComponent } from "./components/quotas/identity/identity-
...
     { path: "dashboard", component: DashboardComponent, canActivate: [AuthGuard] },
     { path: "identities", component: IdentityListComponent, canActivate: [AuthGuard] },
-    { path: "identities/:address", component: IdentityDetailsComponent, canActivate: [AuthGuard] },
+    {
+        path: "identities/:address",
+        component: IdentityDetailsComponent,
+        canActivate: [AuthGuard],
+        data: {
+            reuseComponent: true
+        }
```
Comment (Dannyps) [inline AdminUi/src/AdminUi/ClientApp/src/app/app-routing.module.ts:26]: I'm not familiar with this. I looked it up and I understand the idea, but I'm having a hard time understanding why it's used here. Could you explain?

**Review by Dannyps — DISMISSED** [review]
Looks good 👌

File: AdminUi/src/AdminUi/ClientApp/src/app/app-routing.module.ts (hunk 1 above)
Comment (JLSRKonk) [reply AdminUi/src/AdminUi/ClientApp/src/app/app-routing.module.ts:26]: When we want to link components we generally use something like this:
`await this.router.navigate(["/identities/${identityAddress}"]);`
Which for the most part is fine, but when we want to navigate inside the same component, but with different parameters, using the default router.navigate logic, the current page status is not reset.

We could manually reset the page and run all the methods again, but it becomes tricky when we are also dealing with other imbedded components (Relationships, Messages, etc..), so I figured the best way to handle this is to just make this particular page reload when we navigate with different parameters.

Comment (Dannyps) [reply AdminUi/src/AdminUi/ClientApp/src/app/app-routing.module.ts:26]: Got it, thanks!

File: AdminUi/src/AdminUi/ClientApp/src/app/services/message-service/message.service.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        const httpOptions = {
+            params: new HttpParams()
+                .set("Participant", address)
+                .set("Type", "Outgoing")
+                .set("PageNumber", pageNumber + 1)
+                .set("PageSize", pageSize)
+        };
+
+        return this.http.get<PagedHttpResponseEnvelope<MessageOverview>>(this.apiUrl, httpOptions);
+    }
```
Comment (daniel-almeida-konkconsulting) [inline AdminUi/src/AdminUi/ClientApp/src/app/services/message-service/message.service.ts:39]: Can't this just be a single method given that only the `Type` changes?

**Review by daniel-almeida-konkconsulting — CHANGES_REQUESTED** [review]

**Comment by daniel-almeida-konkconsulting** [comment]
Where are the integration tests?

**Comment by tnotheis** [comment]
I just noticed that there are 3 problems in the UI:
1. When there are more than 3 recipients, the first three recipients should be shown, and **additionally** there should be a button that allows the user to open a popup that shows all recipients. But currently only the button is shown.
2. I think that the button should just have a plus-sign instead of the text "Show Recipients".
3. The popup is pretty wide, so there is a lot of empty space. I think it should be way smaller.

**Comment by tnotheis** [comment]
@HunorTotBagi 
1. Can you resolve the conflicts in this branch please?
2. The formatting pipeline failed. Can you fix it?

**Review by tnotheis — CHANGES_REQUESTED** [review]
There are compiler errors and code that should be removed. I will review once this is fixed.

File: AdminApi/src/AdminApi/ClientApp/package.json [hunk 3]
```diff
@@ -57,6 +57,7 @@
         "typescript": "~5.4.3"
     },
     "overrides": {
-        "autoprefixer": "10.4.5"
+        "autoprefixer": "10.4.5",
+        "vite": "5.1.7"
```
Comment (tnotheis) [inline AdminApi/src/AdminApi/ClientApp/package.json:61]: Why do we need `vite` now? I don't see any references

File: Modules/Messages/src/Messages.Domain/Messages.Domain.csproj [hunk 4]
```diff
@@ -1,6 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <ItemGroup>
+	<PropertyGroup>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
```
Comment (tnotheis) [inline Modules/Messages/src/Messages.Domain/Messages.Domain.csproj:5]: Can be removed. We set this via Directory.Build.props

**Review by tnotheis — CHANGES_REQUESTED** [review]

File: Modules/Messages/src/Messages.Domain/Messages.Domain.csproj (hunk 4 above)
Comment (HunorTotBagi) [reply Modules/Messages/src/Messages.Domain/Messages.Domain.csproj]: Changes made.

**Review by tnotheis — APPROVED** [review]

**Review by daniel-almeida-konkconsulting — APPROVED** [review]

**Pull request merged**